src/group.c

```diff
@@ -2502,6 +2502,14 @@ DYNAMIC_API PEP_STATUS group_rating(
 ) {
 
     PEP_STATUS status = PEP_STATUS_OK;
+    bool active = false;
```

Comment:
In the spirit of defensive coding, I would do it the other way around:  Set the out parameter `rating`  to `PEP_rating_undefined`, and only then to something else if the code below successful determines it. I know, the caller should not use it if the status is not ok, but ...

Reply:
Done, please check.